lib/dbservice/group_users.ex

```diff
@@ -37,6 +37,19 @@ defmodule Dbservice.GroupUsers do
   """
   def get_group_user!(id), do: Repo.get!(GroupUser, id)
 
+  @doc """
+  Gets a group-user based on user_id and group_id.
+  Raises `Ecto.NoResultsError` if the GroupUser does not exist.
+  ## Examples
+      iex> get_group_user_by_user_id(1, 2)
+      %GroupUser{}
+      iex> get_group_user_by_user_id(abc)
+      ** (Ecto.NoResultsError)
+  """
+  def get_group_user_by_user_id(user_id, group_id) do
```

Comment:
We are fetching group_user using `group_id` as well. Can the function name be something appropriate?